recipes/ingrid/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "INGRID" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1d92d669aabb72f2a83763c28a8ad682cc530ae47f11e7131afeda025efd286c
+  patches:
+    - setup.py.patch
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6,<4
+    - pip
+    - setuptools
+  run:
+    - python >=3.6,<4
+    - numpy
+    - sympy
+    - m2r2
+    - scipy >=1.3.1
+    - matplotlib-base
+    - pyyaml
+    - tk
+
+test:
+  imports:
+    - INGRID
```

Comment:
```suggestion
    - INGRID
  commands:
    - pip check
  requires:
    - pip
```

Reply:
@chrisburr Thanks for the review.  Despite listing `tk` as a conda dependency, `pip check` uses a different method of checking for the `tk` package, and indeed `pip install tk` does install something.  So I disabled the `pip check`.  Should I put a patch on the `setup.py` file instead to remove the `tk` from the dependencies?

Reply:
I think `tk` in the `setup.py` dependencies isn't installing what you think it is:
* https://github.com/LLNL/INGRID/blob/78e0f278effd37b6860032e830344041ed5702e1/setup.py#L55
* https://pypi.org/project/tk/

I guess you want https://docs.python.org/3/library/tkinter.html which is part of the standard library.